base/applications/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ add_subdirectory(network)
 add_subdirectory(notepad)
 add_subdirectory(osk)
 add_subdirectory(rapps)
+add_subdirectory(rapps_com)
```

Comment:
Idea: Could you put this stub console shim as a subdirectory of rapps instead? (for self-containedness)

Reply:
Not sure if that's an improvement, the applications folder already has quite some arbitrary folders (cmdutils, screensavers, ..?)
And this will only make it harder to find stuff